packages/expo-system-ui/src/SystemUI.ts

```diff
@@ -4,16 +4,21 @@ import ExpoSystemUI from './ExpoSystemUI';
 
 /**
  * Changes the root view background color.
+ * Call this function in the root file outside of you component.
  *
  * @example
  * ```ts
- * SystemUI.setBackgroundColorAsync("white");
+ * SystemUI.setBackgroundColorAsync("black");
  * ```
  * @param color Any valid [CSS 3 (SVG) color](http://www.w3.org/TR/css3-color/#svg-color).
  */
-export async function setBackgroundColorAsync(color: ColorValue): Promise<void> {
-  const colorNumber = Platform.OS === 'web' ? color : processColor(color);
-  return await ExpoSystemUI.setBackgroundColorAsync(colorNumber);
+export async function setBackgroundColorAsync(color: ColorValue | null): Promise<void> {
+  if (color === null) {
```

Comment:
Don't use `===` for null checks, `==` is better as it handles `undefined` too.
```suggestion
  if (color == null) {
```